sunpy/net/helioviewer.py

```diff
@@ -54,40 +90,6 @@ def get_data_sources(self, **kwargs):
         params = {"action": "getDataSources"}
         params.update(**kwargs)
         return self._get_json(params)
-    
-    def get_source_id(self):
```

Comment:
What was wrong with `get_source_id`?

Reply:
We do not need the function anymore because it wastes time if we make a dictionary everytime we want to get source ID so I initialized a dictionary at the moment when helioviewer is called and then we can extract sourceID any time

Reply:
Thats fair.